### PR TITLE
Split tutorial into two: write and read with scripts

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site link](https://review.docs.microsoft.com/en-us/office/dev/scripts/tutorials/excel-tutorial?branch=AlexJ-SplitTutorial)

The current tutorial was a little long and used two separate examples. This PR divides the tutorial into two parts. The first focuses on using the Action Recorder to make a basic script and working with that script to perform further workbook write operations. The second tutorial focuses on reading data from the workbook before making changes.

Please note there is little net-new content. Only the intros and outros of the tutorials have changed.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5ca7f66](https://github.com/OfficeDev/office-scripts-docs/commits/5ca7f66504affa3edd5ab55c6f5a3e4dc3973a3e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-read-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/tutorials/excel-read-tutorial.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C27%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001271740538529-82%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zy36%2FVllB4J%2FP8kpw6sHVc3EUTKZPdf9uS6MWFhq7AI%3D&st=2020-01-27T17%3A37%3A30Z&se=2020-02-27T17%3A42%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-read-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+3. Run the script.
+
+    Your banking statement should now look like this:
+
+    ![The banking statement as a formatted table with only positive values.](../images/tutorial-5.png)
+
+## Next steps
+
+The [Code Editor](https://aka.ms/office-scripting-overview) has more samples for you to explore as you learn Office Scripts. Visit [Scripting Fundamentals for Office Scripts in Excel on the web](../develop/scripting-fundamentals.md) to learn more about the platform.
```
[the remark this answers is not in the document]
Comment (AlexJerabek) [reply docs/tutorials/excel-read-tutorial.md]: Both the tutorials under only have a "Next Steps" section. To me, that feels like a targeted "See also", so adding more would be a bit redundant. Though, I can be persuaded to add that section :-)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b0cd03e](https://github.com/OfficeDev/office-scripts-docs/commits/b0cd03e9a4b171a936b40d17b387d2d4e90975be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-read-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/tutorials/excel-read-tutorial.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-SplitTutorial/docs/tutorials/excel-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C27%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202001272348556941-82%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yIN%2F7Z1KTA3CrPf5ElnTa1MmpTdctpqVHGuGiGdV2OI%3D&st=2020-01-27T23%3A45%3A06Z&se=2020-02-27T23%3A50%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-read-tutorial.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/tutorials/excel-read-tutorial.md]: That's a fair point...consider me persuaded that a "See also" is unnecessary :)

**Pull request merged**